bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/gateway.xml

```diff
@@ -69,7 +69,6 @@
 		<category>Moisture</category>
 		<tags>
 			<tag>Measurement</tag>
```

Comment:
If no property tag, this one should also be removed.

Reply:
Thanks for keeping me honest. :)